ladybug_display_schema/graphic.py

```diff
@@ -0,0 +1,294 @@
+    horizontal_infrared_radiation_intensity = 'HorizontalInfraredRadiationIntensity'
+    humidity_ratio = 'HumidityRatio'
+    illuminance = 'Illuminance'
+    irradiance = 'Irradiance'
+    liquid_precipitation_depth = 'LiquidPrecipitationDepth'
+    liquid_precipitation_quantity = 'LiquidPrecipitationQuantity'
+    luminance = 'Luminance'
+    mass = 'Mass'
+    mass_flow_rate = 'MassFlowRate'
+    mean_radiant_temperature = 'MeanRadiantTemperature'
+    metabolic_rate = 'MetabolicRate'
+    opaque_sky_cover = 'OpaqueSkyCover'
+    operative_temperature = 'OperativeTemperature'
+    operative_temperature_delta = 'OperativeTemperatureDelta'
+    percentage_people_dissatisfied = 'PercentagePeopleDissatisfied'
+    power = 'Power'
+    precipitable_water = 'PrecipitableWater'
+    predicted_mean_vote = 'PredictedMeanVote'
+    pressure = 'Pressure'
+    prevailing_outdoor_temperature = 'PrevailingOutdoorTemperature'
+    r_value = 'RValue'
+    radiant_coefficient = 'RadiantCoefficient'
+    radiant_temperature = 'RadiantTemperature'
+    radiant_temperature_delta = 'RadiantTemperatureDelta'
+    radiation = 'Radiation'
+    relative_humidity = 'RelativeHumidity'
+    sky_temperature = 'SkyTemperature'
+    snow_depth = 'SnowDepth'
+    specific_energy = 'SpecificEnergy'
+    speed = 'Speed'
+    standard_effective_temperature = 'StandardEffectiveTemperature'
+    temperature = 'Temperature'
+    temperature_delta = 'TemperatureDelta'
+    temperature_time = 'TemperatureTime'
+    thermal_comfort = 'ThermalComfort'
+    thermal_condition = 'ThermalCondition'
+    thermal_condition_eleven_point = 'ThermalConditionElevenPoint'
+    thermal_condition_five_point = 'ThermalConditionFivePoint'
+    thermal_condition_nine_point = 'ThermalConditionNinePoint'
+    thermal_condition_seven_point = 'ThermalConditionSevenPoint'
+    time = 'Time'
+    total_sky_cover = 'TotalSkyCover'
+    utci_category = 'UTCICategory'
+    u_value = 'UValue'
+    universal_thermal_climate_index = 'UniversalThermalClimateIndex'
+    visibility = 'Visibility'
+    voltage = 'Voltage'
+    volume = 'Volume'
+    volume_flow_rate = 'VolumeFlowRate'
+    volume_flow_rate_intensity = 'VolumeFlowRateIntensity'
+    wet_bulb_temperature = 'WetBulbTemperature'
+    wind_direction = 'WindDirection'
+    wind_speed = 'WindSpeed'
+    zenith_luminance = 'ZenithLuminance'
+
+
+class DataTypeBase(BaseModel):
+    """Data type representation."""
+
+    type: constr(regex='^DataTypeBase$') = 'DataTypeBase'
+
+    data_type: DataTypes = Field(
+        ...,
+        description='Text to indicate the type of data. This governs the behavior '
+        'of the data type and the acceptable units. The DataTypes enumeration '
+        'contains all acceptable types.'
+    )
+
+    name: str = Field(
+        ...,
+        description='Text to indicate how the data type displays. This can be more '
+        'specific than the data_type.'
+    )
+
+
+class GraphicContainer(BaseModel):
```

Comment:
I think we need two validators here:

1. Check the match between values and geometry.
2. Check that either geometry or (min and max) are being provided.